src/js/providers/utils/network-error-parser.js

```diff
@@ -0,0 +1,33 @@
+export default function parseNetworkError(baseCode, statusCode, url = '', responseText) {
+    let message;
+    let code = baseCode + 1000;
+
+    if (statusCode > 0) {
+        message = badStatusMessage(statusCode, responseText);
+        code += statusCode;
```

Comment:
if `statusCode` is out of the 400-600 range we want to assign it a `010` code. I know it's a very unlikely scenario but I want to ensure our codes never conflict.

Reply:
ok